tests/integration/create_token.py

```diff
@@ -37,13 +38,18 @@ def create_token(form_type_id, eq_id, start_date=None, end_date=None, employment
         else:
             ref_p_employment_date = employment_date
 
+        if region_code is None:
```

Comment:
We could tidy the conditional statements up and put the defaults as the constants possibly